android-components/components/concept/engine/src/main/java/mozilla/components/concept/engine/webextension/WebExtension.kt

```diff
@@ -516,6 +522,41 @@ class DisabledFlags internal constructor(val value: Int) {
     fun contains(flag: Int) = (value and flag) != 0
 }
 
+/**
+ * Incognito values that control how an extension works with private browsing windows.
+ */
+enum class Incognito {
+    /**
+     * The extension will see events from private and non-private windows and tabs.
+     */
+    SPANNING,
+
+    /**
+     * The extension will be split between private and non-private windows.
+     */
+    SPLIT,
+
+    /**
+     * Private tabs and windows are invisible to the extension.
+     */
+    NOT_ALLOWED,
+
+    ;
```

Comment:
nit: I think the `;` should be at the end of `NOT_ALLOWED;`

Reply:
It was like this and then the linter changed that...

Reply:
Yeah, the linter pass, but it looks really odd 😅